components/PicklistBase/PicklistBase.content.comp.cy.js

```diff
@@ -18,11 +18,230 @@ describe('Test the default PicklistBase content', () => {
       props: {
         onReady: readySpy,
         rows: [],
+        columns: ['name', 'quantity', 'location'],
+        labels: { name: 'Name', quantity: 'Quantity', location: 'Location' },
       },
     });
 
     cy.get('@readySpy')
       .should('have.been.calledOnce')
-      .then(() => {});
+      .then(() => {
+        cy.get('[data-cy="picklist-table"]').should('exist');
+        cy.get('[data-cy="picklist-header-name"]').should('exist');
```

Comment:
In addition to checking that these exist this test should be checking the contents as well.  That will catch errors such as having a mismatch between the `data-cy` value and the actual label placed in the button, which an existence test would not catch.